src/js/engagement_view/src/components/reusableComponents/collapsibleDrawer.tsx

```diff
@@ -101,59 +91,17 @@ export default function CollapsibleNavDrawer() {
                             )}
                         </IconButton>
                         <Icon>
-                            <img className={classes.logoImage} src={Img} />
+                            <img
+                                className={classes.logoImage}
+                                src={Img}
+                                alt="Grapl Logo"
+                            />
```

Comment:
Looks like we have a number of these - perhaps we should make this its own `<GraplLogo />` component?

![image](https://user-images.githubusercontent.com/69007229/167709519-762ef7b1-49e2-4a3d-9323-adc73ce15dcc.png)

Reply:
Done. Good call.